ci-operator/jobs/openshift/cluster-api-provider-gcp/openshift-cluster-api-provider-gcp-master-presubmits.yaml

```diff
@@ -36,7 +36,8 @@ presubmits:
         - name: JOB_NAME_SAFE
           value: e2e-gcp-operator
         - name: TEST_COMMAND
-          value: make test-e2e
+          value: cd /go/src/github.com/openshift/cluster-api-provider-gcp && make
+            test-e2e
```

Comment:
any reason this one is on a separate line?

Reply:
This line was generated. Lemme run the generator again.

Reply:
Used multi-line command instead.